bats/fb-katello-content.bats

```diff
@@ -181,6 +181,11 @@ setup() {
   # /var/lib/pulp/exports/Test_Organization/Test_CV/1.0/2020-12-11T16-04-08-00-00
   import_path="/var/lib/pulp/imports/bats-test-$export_history_id"
 
+  # assert latest_export, export_path and export_history_id aren't empty
+  [ -n "${latest_export}" ]
```

Comment:
Should this happen before we call `cut` on it?

Also, on line 178 In think the comment is outdated since we pass `--fields`, so we should only get the two.

Reply:
I will update the details later, right now I just want the test to cleanly fail again